src/components/NavBar/atlas.js

```diff
@@ -42,7 +40,13 @@ const AtlasNavBar = (props) => {
           </Nav>
           <Nav>
             <NavItem className="nav-icon px-1">
-              <NavLink rel="noreferrer" target='_blank' href="https://kpmp.org/help">
+              <NavLink onClick={() =>{
```

Comment:
This is not right at all.
The category should be based on which application they are using (Explorer vs Spatial Viewer). Also, this is not a download event. It should be action: "Navigation" and label of "Help" as per the card.